src/mpid/ch4/src/ch4_progress.h

```diff
@@ -13,6 +13,37 @@
 
 #include "ch4_impl.h"
 
+#undef FUNCNAME
+#define FUNCNAME MPID_Progress_test_noinline
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static int MPID_Progress_test_noinline(void)
```

Comment:
I'm concerned about the naming here. This is not simply a non-inline version of `MPID_Progress_test`, but a different function that only makes transport-level progress. That is, it doesn't progress any global lists or schedules. We should re-name it accordingly.

Also, because it is non-inline, do we want to move it to a `.c` file and remove the `static` declaration? Or would that be problematic?

Reply:
It was my first idea, to move it into `.c` file, but I wasn't sure about right placement. `ch4_globals.c` isn't seems to be a right place for this function. We may add new one anyway.